client/src/utils/unitHelpers.ts

```diff
@@ -30,8 +30,9 @@ export function parseUnitsOfRam(bytes?: number) {
     };
 }
 
-function parseUnitsOfBytes(value?: string) {
+function parseUnitsOfBytes(value?: string | {string: string}) {
```

Comment:
I don't see where this change is being used. Is this still needed?

Reply:
the value passed in here is an object {string: "some string"} for proto while it's a string for json. I changed the line below as `value = typeof value === 'object'? value?.string : value;`